xla/service/gpu/hlo_fusion_analysis.cc

```diff
@@ -209,6 +209,10 @@ HloFusionAnalysis::EmitterFusionKind HloFusionAnalysis::GetEmitterFusionKind()
       fusion_backend_config_.kind() == kTritonSoftmaxFusionKind) {
     return EmitterFusionKind::kTriton;
   }
+
+  if (fusion_backend_config_.kind() == kCuDnnFusionKind) {
+    return EmitterFusionKind::kCuDnn;
```

Comment:
FYI for future reference: We're trying to get rid of this enum, so feel free not to add a new value next time. The logic can live right in GetFusionEmitter.

Reply:
Ack. Will leave as is for now for uniformity.